astropy/table/table.py

```diff
@@ -2564,12 +2579,17 @@ class QTable(Table):
         Copy the input data (default=True).
     rows : numpy ndarray, list of lists, optional
         Row-oriented data for table instead of ``data`` argument
+    copy_indices : bool, optional
+        Copy any indices in the input data (default=True)
+    **kwargs : dict, optional
+        Additional keyword args when converting table-like object
 
     """
     def __init__(self, data=None, masked=None, names=None, dtype=None,
```

Comment:
You could remove this whole `__init__`, since all you do is call the super... (but only worry about this if you need to make another change anyway).